app/javascript/components/physical-storage-form/physical-storage-form.schema.js

```diff
@@ -0,0 +1,73 @@
+import { componentTypes, validatorTypes } from '@@ddf';
+
+const loadProviders = () =>
+  API.get(
+    '/api/providers?expand=resources&attributes=id,name,supports_block_storage&filter[]=supports_block_storage=true',
+  ).then(({ resources }) =>
+    resources.map(({ id, name }) => ({ value: id, label: name })));
+
+const loadFamilies = id => API.get(`/api/providers/${id}?attributes=type,physical_storage_families`)
+  // eslint-disable-next-line camelcase
+  .then(({ physical_storage_families }) => physical_storage_families.map(({ name, id }) => ({
+    label: name,
+    value: id,
+  })));
+
+const createSchema = (emsId, setEmsId) => ({
+  fields: [
+    {
+      component: componentTypes.SELECT,
+      name: 'ems_id',
+      key: 'ems_id',
+      id: 'ems_id',
+      label: __('Provider:'),
+      loadOptions: loadProviders,
+      onChange: value => setEmsId(value),
+      validate: [{ type: validatorTypes.REQUIRED }],
+    },
```

Comment:
Please add an `isRequired: true` attribute for each required field. This should display the `*` after the label.